packages/contracts/contracts/plugin/examples/count/CounterV1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
- Shouldn't this be called `metadata.json` as well?
- I would add `name: "...", description: ""` everywhere
- `setupABIs` => `pluginSetupABI:  {prepareInstallation, prepareUpdate, prepareUninstallation}`

Reply:
`name`, `description` + 

putting this dirrectly mightn't work:

```
(tuple(uint8 votingMode, uint64 supportThreshold, uint64 minParticipation, uint64minDuration, uint256 minProposerVotingPower) votingSettings, address[] members
```. Front-end will have a hard time to decode this.

probably, it should have an actual ABI(how it looks normally, or we could do `human readable ABI` from ethers.js.

Needs really good look to make it very nice.

Reply:
@brickpop setupABIs => pluginSetupABI is amended.
@novaknole it is already "full" format human readable abi, or i am missing somthing

Reply:
@novaknole amended in [0bfecf2](https://github.com/aragon/core/pull/208/commits/0bfecf27606444e4c03914913f0c90c19a2dcb69)